tooling/debugger/build.rs

```diff
@@ -70,3 +72,67 @@ fn debug_{test_name}() {{
         .expect("Could not write templated test file.");
     }
 }
+
+fn generate_debugger_test_tests(test_file: &mut File, test_data_dir: &Path) {
```

Comment:
Nit (and matter of taste I guess):

```suggestion
fn generate_test_runner_debugger_tests(test_file: &mut File, test_data_dir: &Path) {
```